### ISSUE TITLE
Documentation doesn't match stable distribution

### ISSUE THREAD

**Issue by slbayer** [mozilla/pdf.js#10889]
You've updated your documentation page to reflect the behavior of the current beta, but the example shown will not work in the stable distribution 2.0.943, because the signature of the getViewport() method has changed. If you ask me, the example documentation should reflect the stable distribution, or, at the very least, note that the behavior has changed.

Attach (recommended) or Link to PDF file here:

Configuration:
- Web browser and its version: N/A
- Operating system and its version: N/A
- PDF.js version: 2.0.943
- Is a browser extension: no

Steps to reproduce the problem:
1. Visit https://mozilla.github.io/pdf.js/examples/
2. Attempt to implement the hello world example.

What is the expected behavior? (add screenshot)

Hello world example should work.

What went wrong? (add screenshot)

getViewport() accepts a JavaScript object in the current HEAD, but a scale value in the stable distribution.

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

**Comment by Snuffleupagus** [comment]
> This really should have been made backward-compatible; it would have been trivial to implement getViewport() to accept the old arguments and the new ones.

As can be seen the API, the *method* itself is backwards compatible: https://github.com/mozilla/pdf.js/blob/63014eb897ba0a1fce210ad89cf8ce5ba98c081a/src/display/api.js#L960-L980

Probably the examples shouldn't have been update when they were, but given that the next release should happen *really* soon (already tracked in #10819), I don't think it's necessary to make any changes at this point in time.

**Comment by timvandermeij** [comment]
Yes, we're planning on making a new release soon as tracked in the issue above, so then this will be fixed automatically.
